### PR TITLE
Update docs to reflect multiple source CVE

### PR DESCRIPTION
**Pull request by baweaver**
As per the issue [bundler#5062](https://github.com/bundler/bundler/issues/5062)

A preliminary draft of some documentation changes. Likely this should be persisted to the default template as well which I can hunt down. I'll be updating this as per request.

### CODE REVIEW

File: source/v1.5/gemfile.html.haml [hunk 1]
```diff
@@ -15,7 +15,11 @@
       Gemfiles require at least one gem source, in the form of the URL for a RubyGems server. Generate a Gemfile with the default rubygems.org source by running <code>bundle init</code>. If you can, use <code>https</code> so your connection to the rubygems.org server will be verified with SSL.
     :code
       # lang: ruby
-      source 'https://rubygems.org'
```
Comment (lynnco) [inline source/v1.5/gemfile.html.haml]: I'm not sure this applies to Bundler versions <1.7

Comment (baweaver) [inline source/v1.5/gemfile.html.haml]: I'll leave that one to you. I have no issue either way.

**Comment by lynnco** [comment]
@baweaver can you change to title to "Update docs to reflect multiple source CVE"

**Comment by baweaver** [comment]
Done

**Comment by lynnco** [comment]
Will let this sit for a bit before merging

**Comment by baweaver** [comment]
Fair fair, let me know if you need anything else on it. I'll see about becoming more familiar with some of the code base later.

**Comment by lynnco** [comment]
Approved!

File: source/blog/2014-08-14-bundler-may-install-gems-from-a-different-source-than-expected-cve-2013-0334.html.markdown [hunk 2]
```diff
@@ -28,7 +28,21 @@ To mitigate this, the Bundler and RubyGems.org teams worked together to copy alm
 
 To resolve this issue, upgrade to Bundler 1.7 by running `gem install bundler`. The next time you run `bundle install` for any Gemfile that contains multiple sources, each gem available from multiple sources will print a warning.
 
-For every warning printed, edit the Gemfile to either specify a `:source` option for that gem, or move the `gem` line into a block that is passed to a `source` method call.
+It is recommended to use a source block to contain all gems intended to be installed from rubygems.org. An example of a vulnerable file would be:
```
Comment (segiddins) [inline source/blog/2014-08-14-bundler-may-install-gems-from-a-different-source-than-expected-cve-2013-0334.html.markdown]: this shouldn't be necessary if there's only a single rubygems source in the Gemfile

File: source/v1.10/gemfile.html.haml [hunk 3]
```diff
@@ -15,9 +15,11 @@
       Gemfiles require at least one gem source, in the form of the URL for a RubyGems server. Generate a Gemfile with the default rubygems.org source by running <code>bundle init</code>. If you can, use <code>https</code> so your connection to the rubygems.org server will be verified with SSL.
     :code
       # lang: ruby
-      source 'https://rubygems.org'
+      source 'https://rubygems.org' do
+        # Gems here
+      end
     .notes
-      It is possible, but not recommended as of Bundler 1.7, to add multiple global source lines. These are searched from last to first.
+      Global source lines are a security risk and should not be used as they can lead to gems being installed from unintended sources.
```
Comment (segiddins) [inline source/v1.10/gemfile.html.haml]: again, only an issue if there are multiple rubygems sources

**Comment by indirect** [comment]
@baweaver are you able to address the comments on this PR in a new PR?

**Comment by baweaver** [comment]
I can update it tomorrow evening. Sorry, been a bit busy.

**Pull request merged**